project/plugins.sbt

```diff
@@ -1,12 +1,17 @@
 resolvers += MavenRepository("HMRC-open-artefacts-maven2", "https://open.artefacts.tax.service.gov.uk/maven2")
 resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.7.0")
+// To resolve a bug with version 2.x.x of the scoverage plugin - https://github.com/sbt/sbt/issues/6997
+ThisBuild / libraryDependencySchemes ++= Seq(
+  "org.scala-lang.modules" %% "scala-xml" % VersionScheme.Always
```

Comment:
Think this is duplicated in `build.sbt`, might want to remove it either here or in the build file?

Reply:
@jpatton638 I know, but it will fail in Jenkins if it is not in 2 places but locally works fine

Reply:
@ashwinirudrappa I see, no worries! You might have accidentally sorted my issue! 😂